src/sdl_wrappers.h

```diff
@@ -2,8 +2,25 @@
 #ifndef SDL_WRAPPERS_H
 #define SDL_WRAPPERS_H
 
-#include <SDL2/SDL.h>
-#include <SDL2/SDL_ttf.h>
+#if defined(__ANDROID__) || defined(OSXCROSS) || (defined(_MSC_VER) && !defined(USE_VCPKG))
+#   include <SDL.h>
```

Comment:
Could this be changed to
```
#if defined(_MSC_VER) && defined(USE_VCPKG)
#    include <SDL2/SDL_xxx.h>
#else
#    include <SDL_xxx.h>
#endif
```
instead? My local build with MinGW-w64 on Windows is not compiling after #26659 because the compiler cannot find the headers. Since all supported builds were working before that change, I think it's best to only special case the new vcpkg build option.

As to why the CI builds are not failing... My guess is that the parent directory of the SDL2 folder just happens to be set as another include path in the CI builds.

Reply:
Yeah, that's much better and should resolve all existing builds. Thanks!